torch/_functorch/_aot_autograd/jit_compile_runtime_wrappers.py

```diff
@@ -385,7 +385,7 @@ class CompiledFunction(torch.autograd.Function):
 
         @staticmethod
         def _compiled_autograd_key(ctx):
-            return (aot_config.aot_id, *ctx.symints)
+            return (aot_config.aot_id, ctx._autograd_function_id, *ctx.symints)
```

Comment:
Args past the first are treated as symints.  I think we can just do 
```suggestion
            return (ctx._autograd_function_id, *ctx.symints)
```